lib/routes/sign.js

```diff
@@ -87,19 +87,23 @@ module.exports = function (log, isA, error, signer, db, domain) {
             })
           }
         }
-        signer.sign(
-          {
-            email: sessionToken.uid.toString('hex') + '@' + domain,
+        var uid = sessionToken.uid.toString('hex')
+        var account
+        db.account(uid).then(function(a) {
```

Comment:
Hmm....I'm a little wary of adding another db read here, given that this is one of more frequently-called endpoints.  An alternative would be to slurp in the account.createdAt when we read the sessionToken, and expose it as a property on the session token in the same way we currently do for `verifierSetAt`, `locale` and others.